core/src/plan/expr/mod.rs

```diff
@@ -116,8 +124,10 @@ mod tests {
 
         // PlanExpr::CompoundIdentifier
         let actual = expr("Foo.id");
-        let expected = ["Foo".to_owned(), "id".to_owned()];
-        let expected = PlanExpr::CompoundIdentifier(&expected);
+        let expected = PlanExpr::CompoundIdentifier {
+            alias: &"Foo".to_string(),
+            ident: &"id".to_string(),
```

Comment:
This is a situation that is caught when the clippy target is tested.
`cargo clippy --release --all-features --all-targets --workspace`

```suggestion
            alias: "Foo",
            ident: "id",
```

Let's change it to